tm-parsers/js/js.tm

```diff
@@ -1963,30 +1995,21 @@ ${end}
 ${template go_parser.lookaheadNext}${end}
 ${template go_parser.callLookaheadNext(memoization)}lookaheadNext(&lexer, end, ${memoization?'nil /*empty stack*/':'stack'})${end}
 
-${template newTemplates-}
```

Comment:
I think we need to revert all changes to templates here. Some of them are needed for the "Textmapper in Go" migration.